src/cluster/common.py

```diff
@@ -213,68 +213,3 @@ def report_results(self, start, jobs_launched, jobs_skipped, jobs_rejected):
 
         self.log.info(msg)
 
-    def handle_all(self, start):
```

Comment:
Why was this moved? I'm assuming Kofalt designed it this way for abstraction/encapsulation.

Reply:
`handle_all` was moved because the functions it references are in the descendant class (Base) and make it harder to test directly.

This is a non-breaking change as demonstrated in tests on `scien-hpc`.